src/sass/app.scss

```diff
@@ -88,80 +109,94 @@ a {
 	padding: 1px 8px;
 }
 
-.panel-notificacion {
+.notification-panel {
 	@include display-flex();
 	@include align-items(center);
 	background-color: #ffffff;
-	width: 50%;
 	border-radius: 8px;
 	padding: 22px;
 	margin: 35px 0px;
 }
 
-.punto-notificacion {
-	@include align-self(start);
-	background-color: #FF2861;
-	width: $anchopunto;
-	height: 16px;
+.image-container { 
+	display: flex;
+	@include flex($width-image-container);
+	position: relative;
+}
+
+.notification-badge {
+	position: absolute;
+	background-color: $primary-color;
+	width: $width-and-height-badge;
+	height: $width-and-height-badge;
 	border-radius: 50%;	
 }
 
-.espacio-imagen {
-	width: $anchoimagen;
-	padding: 23px 0px;
+.image-area {
+	padding: 30px 0px;
+	margin-left: 18px;
 	line-height: 0px;
 }
 
-.imagen-contacto {
+.contact-image {
 	max-width: 100%;
 	max-height: 100%;
 	border-radius: 50%;
 }
 
-.espacio-texto {
+.text-area {
 	text-align: left;
-	width: calc(100% - #{$anchoimagen + $anchopunto});
+	width: calc(100% - #{$width-image-container});
```

Comment:
we're missing the flex property here, you should use it instead of `width`: `flex: 1 1 100%;` here you have a good and complete [flexbox guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/):

![image](https://user-images.githubusercontent.com/662930/38462039-f3aa9d82-3aa4-11e8-8702-0d6747d03dc2.png)